R/Achilles.R

```diff
@@ -78,14 +79,18 @@ achilles <- function (connectionDetails,
                       validateSchema = FALSE,
                       vocabDatabaseSchema = cdmDatabaseSchema,
                       runCostAnalysis = FALSE,
-                      sqlOnly = FALSE){
+                      sqlOnly = FALSE,
+                      conceptHierarchy = TRUE){

```

Comment:
The addition of the `conceptHierarchy` parameter with a default value of `TRUE` is likely why the tests will not run to completion. This is now a mandatory step to create a single table (ConceptHeirarchy) from all of the CDM vocabulary tables which will be leveraged by WebAPI.

I know @chrisknoll has attempted to get the tests to run for a longer period of time to avoid this problem. Assuming we can't do it that way, here is another idea. We could augment the tests to explicitly set `conceptHierarchy = FALSE` to bypass this step. Then, separately, we would test the [ConceptHierarchy_v5.sql](https://github.com/OHDSI/Achilles/blob/7fd682d4944090373dab2797bc0d835bd5a77b92/inst/sql/sql_server/ConceptHierarchy_v5.sql) to make sure that it is working across platforms?

Reply:
I agree.